components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -257,18 +302,6 @@ abstract class AbstractFetchDownloadService : Service() {
         context.registerReceiver(broadcastReceiver, filter)
     }
 
-    private fun displayOngoingDownloadNotification(download: DownloadState) {
-        val ongoingDownloadNotification = DownloadNotification.createOngoingDownloadNotification(
-            context,
-            download
-        )
-
-        NotificationManagerCompat.from(context).notify(
-            downloadJobs[download.id]?.foregroundServiceId ?: 0,
-            ongoingDownloadNotification
-        )
-    }
-
     @Suppress("ComplexCondition")
     internal fun performDownload(download: DownloadState) {
         val isResumingDownload = downloadJobs[download.id]?.currentBytesCopied ?: 0L > 0L
```

Comment:
0L > 0L would always be false, right? 

The logic is: `isResumingDownload` is TRUE if the bytes copied is more than 0. if bytesCopied is null, we assume it's 0, and thus this would evaluate to false.

Reply:
Why evaluate instead of simplifying this to be `false`?

Reply:
I'm confused. I want to compare `currentBytesCopied` to 0L so I can't just do 

`downloadJobs[download.id]?.currentBytesCopied ?: false > 0L`

Is there some syntax I'm missing here? 😅

Reply:
Try this so avoid confusion:
```kotlin
(downloadJobs[download.id]?.currentBytesCopied ?: 0L) > 0L
```

Otherwise, it looks like:
```kotlin
downloadJobs[download.id]?.currentBytesCopied ?: (0L > 0L)
```